cmd/restic/cmd_forget.go

```diff
@@ -80,9 +80,15 @@ func init() {
 	f.BoolVar(&forgetOptions.Prune, "prune", false, "automatically run the 'prune' command if snapshots have been removed")
 
 	f.SortFlags = false
+	addPruneFlags(cmdForget)
```

Comment:
Since it's called options in most other parts of the code, I guess this should be named `addPruneOptions`? :)

Reply:
is changed.